src/plugins/intel_gna/src/backend/gna_limitations.hpp

```diff
@@ -300,14 +302,31 @@ class Limitations {
     constexpr static uint32_t kMemoryPageSize = 4096;
 
 private:
+    struct TargetLimitations {
+        TargetLimitations() = default;
+
+        TargetLimitations(bool buffers_memory,
```

Comment:
Isn't it somehow duplicating  functionally gna_lib_reserves_scratch? From the table it would seem so...

Reply:
Memory buffers are utilized for convolution, and scratch memory is reserved by the GNA library for fully connected layers. From a plugin perspective, it doesn't matter, but two separate fields better reflect the hardware.